packages/bruno-app/src/components/RequestPane/Auth/OAuth2/ClientCredentialsMethodSelector/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { forwardRef, useEffect, useRef, useState } from 'react';
+import Dropdown from 'components/Dropdown';
+import StyledWrapper from './StyledWrapper';
+import { IconCaretDown } from '@tabler/icons';
+import { updateAuth, updateCollectionAuth } from 'providers/ReduxStore/slices/collections';
+import { useDispatch } from 'react-redux';
+import { humanizeOAuth2ClientSecretMethod } from 'utils/collections';
+
+const ClientCredentialsMethodSelector = ({ item, collection, oAuth }) => {
```

Comment:
The same component is used in `RequestPage` and in `CollectionSettings/Auth`. While the convention in the project seems to be using duplicate, almost identical components, I feel this approach is easier to manage. Perhaps it should be moved into a different location then?

Reply:
seconded - perhaps we should have a `shared` components directory